libs/ui-watt/src/lib/components/button/watt-button.component.scss

```diff
@@ -14,109 +14,120 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@use './watt-button-variables' as buttonVariables;
+@use '@energinet-datahub/watt/utils' as watt;
+
+watt-button .mat-button {
+  line-height: 24px;
+  @include watt.typography-font-weight("semi-bold");
+  @include watt.typography-font-size("m");
+  @include watt.space-inset-squish-m;
+  height: 44px; // Magic UX normal height
+  min-width: 44px; // Magic UX width
+
+  &.watt-button--large {
+    height: 56px; // Magic UX large height
+  }
 
-watt-button {
-  display: inline-block;
+  watt-spinner {
+    padding: 3px 26px; // MAgic UX padding for button size
+  }
 
-  &.watt-button-size-normal {
-    button.watt-button {
-      height: buttonVariables.$watt-button-size-normal-height;
-      min-width: buttonVariables.$watt-button-min-width;
-    }
+  .mat-button-focus-overlay {
+    background: transparent; // Remove the overlay background color from Material Design
   }
 
-  &.watt-button-size-large {
-    button.watt-button {
-      height: buttonVariables.$watt-button-size-large-height;
-      min-width: buttonVariables.$watt-button-min-width;
+  .mat-button-wrapper {
+    display: flex;
+    gap: var(--watt-space-s);
+    justify-content: center;
+
+    watt-icon {
+      margin: -5px; // to offset the icons inherent padding of 5px
     }
   }
 
-  &.watt-button-loading {
-    .content-grid {
-      position: relative;
+  &.watt-button--primary {
+    background-color: var(--watt-color-primary);
+    color: var(--watt-color-primary-contrast);
+
+    &:focus {
+      outline: 2px solid var(--watt-color-secondary);
     }
 
-    .content-grid-item-spinner {
-      position: absolute;
+    &:focus-visible {
+      outline: 2px solid red;
     }
 
-    .content-grid-item:not(.content-grid-item-spinner) {
-      visibility: hidden;
+    &:hover,
+    &:focus {
+      background: var(--watt-color-primary-dark);
     }
 
-    watt-primary-button {
-      .mat-spinner circle {
-        stroke: var(--watt-color-neutral-white);
-      }
+    &.mat-button.mat-button-disabled {
+      background-color: transparent;
+      color: var(--watt-color-neutral-grey-500);
     }
-  }
 
-  &.watt-button-disabled {
-    // NOTE(xdzus)
-    // Prevents emitting a click event in Chrome/Edge/Safari when a disabled button is clicked
-    pointer-events: none;
```

Comment:
Yes, this is available to use, but in case we want to use cursor: not-allowed, we might want to make the developer take care of "disabled" state on button clicks instead :)